treeherder/perf/management/commands/synthesize_backfill_report.py

```diff
@@ -0,0 +1,105 @@
+from argparse import ArgumentError
+from datetime import (datetime,
+                      timedelta)
+from typing import (List,
+                    Tuple)
+
+from django.core.management.base import BaseCommand
+from django.db.models.query import QuerySet
+
+from treeherder.perf.alerts import (AlertsPicker,
+                                    IdentifyRetriggerables)
+from treeherder.perf.models import (PerformanceAlertSummary,
+                                    PerformanceFramework)
+
+
+class Command(BaseCommand):
+    repos_to_retrigger_on = ['autoland', 'mozilla-inbound', 'mozilla-beta']
+    help = "Select most relevant alerts and identify jobs to retrigger."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--time-window',
+            action='store',
+            type=int,
+            default=60,
+            help="How far back to look for alerts to retrigger (expressed in minutes)."
+        )
+
+        parser.add_argument(
+            '--frameworks',
+            nargs='+',
+            default=None,
+            help="Defaults to all registered performance frameworks."
+        )
+
+        parser.add_argument(
+            '--repositories',
+            nargs='+',
+            default=Command.repos_to_retrigger_on,
+            help=f"Defaults to {Command.repos_to_retrigger_on}."
+        )
+
+    def handle(self, *args, **options):
+        frameworks, repositories, max_timestamp, days_number = self._parse_args(**options)
+        self._validate_args(frameworks, repositories)
+
+        summaries_to_retrigger = self._fetch_by(
+            self._summaries_since(max_timestamp),
+            frameworks,
+            repositories
+        )
+        json_output = self._identify_retriggerables(summaries_to_retrigger, days_number)
+
+        print(json_output)
```

Comment:
Should this only print if a `test` flag is passed, rather than doing this as the default?

Reply:
You mean use `test` as a `--debug` like flag for the command?
Given that this command is more of a POC, ideal for a demo we're planning in the next 2-3 weeks, I'd say printing to stdout by default is desired.